tensorflow/contrib/spectral/python/ops/shape_ops.py

```diff
@@ -0,0 +1,52 @@
+import math
```

Comment:
add the copyright header here

see other contrib/ packages for examples of boilerplate you need:

1) a BUILD file
2) an __init__.py file
3) at least one file in python/ops/ (this one)
4) at least one file in python/kernel_tests/ (e.g. shape_ops_test.py) containing unit tests.